### ISSUE TITLE
Git actions not performed when .git folder is not in current working directory

### ISSUE THREAD

**Issue by joerideg** [release-it/release-it#517]
Hi! 
Still using this amazing module, keep up the great work.

I have the following situation:
I have a git repo with a subfolder where my npm package lives that I want to release.
```
- my-repo/
  - .git/
  - mypackage1/
    - package.json
  - mypackage2/
    - package.json
```
now when I am in package1 and do `release-it` it will perform the bump, and ask to publish to npm, but it wont perform the git commit/tag/push. I believe its because of this lookup of the `.git` folder [here](https://github.com/release-it/release-it/blob/7dc2104b69a8a08c34bd06a5e6ab4839df820f29/lib/plugin/git/Git.js#L18)

It would be nice to have the config option to point to the git root dir. What do you think?

Example repo: https://github.com/joerideg/release-it-test

**Comment by webpro** [comment]
Thanks, @joerideg! It was a regression from v11 so I just fixed it in [v11.0.2](https://github.com/release-it/release-it/releases/tag/11.0.2).

**Comment by joerideg** [comment]
Awesome! Thanks alot!
